tutorials/Docking/README.md

```diff
@@ -6,9 +6,11 @@ This tutorial teaches the basics of molecular docking.
 
 Here we dock small molecules indinavir and nelfinavir into HIV-1 protease.
 
-## Requirements
+## Getting started
+
+Please see [https://bigginlab.github.io/OxCompBio/docking.html](https://bigginlab.github.io/OxCompBio/docking.html) for the tutorial instructions, including installation instructions for the software below.
```

Comment:
If possible can you give them a general idea of which bits they have to do?

Essentially the point on Thursday to go "ok so to get started, go to this part in github and do this" and then you have this instruction that goes "do sections A to B of the tutorial".

And then you have a second section that goes "ok now activate your conda environment and resume from here".

Reply:
Note; this is why I went with `setup.md` files so you could have some instructions that are a bit more tailed to "getting stuff ready before the course".